### DISCUSSION TITLE
Friendly Name Energy Dashboard

### DISCUSSION THREAD

**Discussion by Yachtmed** [home-assistant/frontend#20024] [Dashboard]
Any way of adding a Name to individual sensors for 'Monitor Individual Devices' Eg Sockets instead of 
Shelly EM 2 Energy Total 0 ?
Perhaps add to the entity settings as well.

**Comment by karwosts** [comment]
Custom names for individual energy devices merged for 2024.5 ([#20033](https://github.com/home-assistant/frontend/pull/20033))
